src/algebra/order/floor.lean

```diff
@@ -647,6 +634,20 @@ by { ext, simp [le_floor] }
 
 end int
 
+variables [linear_ordered_ring α] [floor_ring α] {a : α}
```

Comment:
This seems pretty dangerous! They were at the correct place before. If you want, you can move the `A floor ring as a floor semiring` section outside the `int` namespace. I'd prefer that.

Reply:
Ah I thought this was what you intended, I agree it's dangerous.

Reply:
I guess this is still TODO?